sentry/src/main/java/io/sentry/protocol/MeasurementValue.java

```diff
@@ -19,6 +19,14 @@
 @ApiStatus.Internal
 public final class MeasurementValue implements JsonUnknown, JsonSerializable {
 
+  public static final String KEY_APP_START_COLD = "app_start_cold";
+  public static final String KEY_APP_START_WARM = "app_start_warm";
+  public static final String KEY_FRAMES_TOTAL = "frames_total";
+  public static final String KEY_FRAMES_SLOW = "frames_slow";
+  public static final String KEY_FRAMES_FROZEN = "frames_frozen";
+  public static final String KEY_TIME_TO_INITIAL_DISPLAY = "time-to-initial-display";
```

Comment:
now when I look at it, should we align with other metrics and use underscores instead of dashes for ttid/ttfd as well?

Reply:
it makes sense, I'll talk to @wmak to update them in the docs and the backend

Reply:
btw, I already did underscores in [relay](https://github.com/getsentry/relay/pull/1943/files#diff-4be114daeb9052778ae699ad5942a07456dd5b519f04d4528404eb2c5a5a2944R323) & [sentry](https://github.com/getsentry/sentry/pull/45876/files#diff-5e30755537fcfd5687d4399272065e49affcc12a4cf7a0bc377cff5234b8a08aR542)

Reply:
yeah i saw it 😅